packages/@webex/plugin-meetings/src/reconnection-manager/index.ts

```diff
@@ -231,23 +231,23 @@ export default class ReconnectionManager {
   }
 
   /**
-   * Stop the local share track.
+   * Stop the local share stream.
    *
    * @param {string} reason a {@link SHARE_STOPPED_REASON}
    * @returns {undefined}
    * @private
    * @memberof ReconnectionManager
    */
-  private async stopLocalShareTrack(reason: string) {
-    await this.meeting.unpublishTracks([
-      this.meeting.mediaProperties.shareVideoTrack,
-      this.meeting.mediaProperties.shareAudioTrack,
+  private async stopLocalShareStream(reason: string) {
```

Comment:
has this file been exempted from UT?

Reply:
No. We have a UT for this file: [packages/@webex/plugin-meetings/test/unit/spec/reconnection-manager/index.js](https://github.com/mkesavan13/webex-js-sdk/blob/wcme3-update-importExports-variablesMethods/packages/%40webex/plugin-meetings/test/unit/spec/reconnection-manager/index.js)

This method will be called internally from the `reconnect` method. So we don't have to change anything in the UT